Dockerfile-linux

```diff
@@ -24,7 +24,6 @@ RUN \
 
 RUN \ 
   curl -fsSL "https://zlib.net/zlib-${zlib_version}.tar.gz" -o zlib-$zlib_version.tar.gz && \
-  shasum -a 256 zlib-$zlib_version.tar.gz && \
```

Comment:
why are we removing this in last commit?

Reply:
these lines seem unnecessary since they just print the shasum, but the next line actually checks them.

Reply:
got it and another docker instance doesn't have there too